jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/SimpleValueReader.java

```diff
@@ -328,6 +343,190 @@ protected int[] _readIntArray(JsonParser p) throws IOException {
         return builder.build().toArray();
     }
 
+    protected long[] _readLongArray(JsonParser p) throws IOException {
+        if (JsonToken.START_ARRAY.equals(p.currentToken())) {
+            p.nextToken();
+        }
+
+        java.util.List<Long> values = new java.util.ArrayList<>();
+        int t = p.currentTokenId();
+
+        if (t == JsonTokenId.ID_END_ARRAY) {
+            return NO_LONGS;
+        }
+        
+        main_loop:
+        while (true) {
+            switch (t) {
+            case JsonTokenId.ID_NUMBER_FLOAT:
+            case JsonTokenId.ID_NUMBER_INT:
+            case JsonTokenId.ID_NULL:
+                values.add(p.getValueAsLong());
+                break;
+            case JsonTokenId.ID_END_ARRAY:
+                break main_loop;
+            default:
+                throw new JSONObjectException("Failed to bind `long` element if `long[]` from value: "+
```

Comment:
Funny. Copied typo from `int[]` case :)

("if" -> "of")

(will fix separately)